zfs/zfs.go

```diff
@@ -329,10 +330,30 @@ func (a ZFSSendArgsUnvalidated) buildCommonSendArgs() ([]string, error) {
 		return args, nil
 	}
 
-	if a.Encrypted.B {
+	if (a.Encrypted != nil && a.Encrypted.B) || a.Raw {
```

Comment:
a.Encrypted must always be not nil, it's a logic error in caller if` a.Encrypted == nil`.
I'll push a commit to this branch that explains the `NilBool` in a doc comment.